packages/utils/src/retry.ts

```diff
@@ -61,7 +61,7 @@ const retry = <TParams extends unknown[], TReturn>(
             setTimeout(
               resolve,
               retryDelay +
-                Math.min(maxExponentialBackoffDelay, exponentialBackoffDelay * 2 ** retry),
+                Math.min(maxExponentialBackoffDelay, (exponentialBackoffDelay / 2) * 2 ** retry),
```

Comment:
This change adjusts the retry logic to make `exponentialBackoffDelay` more intuitive to configure
* Now, 1000 means retry after 1, 2, 4, 8... seconds.
* Previously, 1000 meant retry after 2, 4, 8, 16... seconds.